manifests/t/Tencent/TencentMeeting/3.11.7.422/Tencent.TencentMeeting.locale.en-US.yaml

```diff
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Tencent.TencentMeeting
+PackageVersion: 3.11.7.422
+PackageLocale: en-US
+Publisher: Tencent Technology (Shenzhen) Co. Ltd.
+PublisherUrl: https://meeting.tencent.com/
+PublisherSupportUrl: https://meeting.tencent.com/support/
+PrivacyUrl: https://meeting.tencent.com/privacy-policy-en.html
+Author: Shenzhen Tencent Computer Systems Company Limited
+PackageName: Tencent Meeting
+PackageUrl: https://meeting.tencent.com/download-center.html
+License: Proprietary
+LicenseUrl: https://meeting.tencent.com/declare-en.html
+Copyright: Copyright © 2018 - 2022 Tencent Inc. All Rights Reserved.
+CopyrightUrl: https://www.tencent.com/en-us/statement.html
+ShortDescription: Boundless Collaboration
+Description: Backed by Tencent's 21 years of experience in audiovisual communications and Tencent Cloud's global network deployment, Tencent Meeting provides smooth, secure, and reliable cloud-based HD conferencing services that enable you to start or join meetings with ease anytime, anywhere. It features online document collaboration, join via WeChat Mini Programs, meeting management and control, screen sharing and many more. With Tencent Meeting, mobile working and cross-company meetings are no longer a problem!
+Moniker: wemeet
+Tags:
+- conference
+- conferencing
+- meeting
+- video-conferencing
+- voice-conferencing
+# Agreements:
+ReleaseNotes: |-
+  【防入侵模式】
+  会议主持人可以一键控制会议成员音视频、屏幕共享、批注、聊天等权限，快速控场确保会议秩序。
+  【嘉宾名片】
+  网络研讨会支持设置嘉宾名片。
+  【嘉宾邀请函】
+  网络研讨会优化嘉宾邀请函，邀请流程更有仪式感。
+  【会聚模式】
+  新增礼堂、露营地等12种场景素材。
+ReleaseNotesUrl: https://meeting.tencent.com/support.html?tab=42
+PurchaseUrl: https://meeting.tencent.com/buy/
+# InstallationNotes:
+# Documentations:
+ManifestType: defaultLocale
```

Comment:
seems like zh-CN should be the defaultLocale

Reply:
I choose `en-US` as defaultLocale so that users from regions other than en-US or zh-CN will eventually receive en-US results when using `winget show`.